crossenv/__init__.py

```diff
@@ -808,6 +816,11 @@ def main():
                 If not given, an attempt will be made to guess. This is used
                 to trick some packages into finding required headers and is
                 optional.""")
+    parser.add_argument('--sysconfigdata-file', action='store',
+        help="""Explicitly set the sysconfigdata file path.
+                If not given, all sysconfigdata files will be searched and
+                will error if there are multiple files that have different
+                values.""")
```

Comment:
Can you mention that this is a workaround specific to Anaconda? (As far as I know, anyway.) I think for most users of crossenv, if they get an error about multiple sysconfigdata files, it indicates some other problem. I don't want to send anyone down the wrong path.